torch/distributed/rpc/api.py

```diff
@@ -28,26 +28,43 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
-def join_rpc():
+def wait_all_workers():
     r"""
     Block until all local and remote RPC processes reach this method, process
```

Comment:
Remove the part which says: "process all pending messages". This is true today for the ProcessGroup backend, but would not necessarily be true for other backends.